h/claim/__init__.py

```diff
@@ -0,0 +1,13 @@
+from webob.cookies import SignedSerializer
+
+
+def includeme(config):
+    config.include('.views')
+    config.include('.util')
+
+    # We use the shared session secret, but salt it with the namespace
+    # 'h.claim' -- only messages serialized with this salt will
+    # authenticate on deserialization.
+    secret = config.registry.settings['session.secret']
+    serializer = SignedSerializer(secret, 'h.claim')
+    config.registry.claim_serializer = serializer
```

Comment:
Actually, the auth ticket policy doesn't use a salt. The session does. It's docs say:

    ``salt``
      A namespace to avoid collisions between different uses of a shared
      secret. Reusing a secret for different parts of an application is
      strongly discouraged (see :ref:`admonishment_against_secret_sharing`).
      Default: ``'pyramid.session.'``.